From d9f280e7041590bbfcd456ad39556e0c52252731 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Wed, 30 Jan 2019 14:47:46 +0800
Subject: [PATCH] axxiaarm: change value of AXXIA_DEBUG_UART_VIRT

The macro AXXIA_DEBUG_UART_VIRT is defined to be 0xf0080000 which
is not in vmalloc zone (0xf0800000-0xff800000), and it will trigger
warning as below:

BUG: mapping for 0x2010080000 at 0xf0080000 out of vmalloc space

So change the value to be 0xf0880000 to avoid this warning.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm/mach-axxia/include/mach/hardware.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-axxia/include/mach/hardware.h b/arch/arm/mach-axxia/include/mach/hardware.h
index 38c7577..4015d2e 100644
--- a/arch/arm/mach-axxia/include/mach/hardware.h
+++ b/arch/arm/mach-axxia/include/mach/hardware.h
@@ -17,7 +17,7 @@
 #define AXXIA_UART3_PHYS       0x2010083000
 
 #ifdef CONFIG_DEBUG_LL_AXXIA_UART0
-#define AXXIA_DEBUG_UART_VIRT  0xf0080000
+#define AXXIA_DEBUG_UART_VIRT  0xf0880000
 #define AXXIA_DEBUG_UART_PHYS  AXXIA_UART0_PHYS
 #endif
 
-- 
1.7.9.5

